### PR TITLE
track source metadata for authoring projections

### LINKED ISSUE #371: Project WinMD assembly information in consuming projection
In order to be able to author a component in C# and then produce a WinMD from it, you need to be able to resolve all the types from a projection referenced by a C# component back to its WinMD assembly.  To do that, this information first needs to be projected.

### PR DESCRIPTION
**Pull request by Scottj1s**
fixes #371 

Jeremy - any concern with piggybacking the source metadata (sans path, extension) onto WindowsRuntimeType for authoring support?  trying to economize on attributes

### CODE REVIEW

File: Strings.props [hunk 1]
```diff
@@ -4,6 +4,9 @@
     <PrepareForBuildDependsOn>
       UpdateStringSources;UpdateStringsHeader;$(PrepareForBuildDependsOn)
     </PrepareForBuildDependsOn>
+    <ComputeCompileInputsTargets>
```
Comment (Scottj1s) [inline Strings.props:7]: this is just to hide the glut of generated string sources in the cswinrt project node (the previous technique stopped working with a VS update)

File: WinRT.Runtime/Projections/Type.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 namespace ABI.System
 {
-    [WindowsRuntimeType]
+    [WindowsRuntimeType("Windows.UI.Xaml")]
```
Comment (manodasanW) [inline WinRT.Runtime/Projections/Type.cs:8]: Is this accurate?

Comment (Scottj1s) [reply WinRT.Runtime/Projections/Type.cs:8]: oops, meant to fix that - should be Windows.UI.Xaml.Interop

Comment (manodasanW) [reply WinRT.Runtime/Projections/Type.cs:8]: This should be the assembly right not the namespace?

Comment (Scottj1s) [reply WinRT.Runtime/Projections/Type.cs:8]: oh, had it right to begin with - yes, the assembly.  e.g., C:\Windows\System32\WinMetadata\Windows.UI.Xaml.winmd

**Pull request merged**